docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives).
+
+In the event that the Primary Domain Validation Determination or Primary CAA Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table, CAs MAY immediately retry Multi-Perspective Domain Validation using the same validation method or an alternative method (e.g., a CA can immediately retry validation using “Email to DNS TXT Contact” if “Agreed-Upon Change to Website - ACME” does not corroborate the outcome of Multi-Perspective Domain Validation). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+Phased Implementation Timeline:
+- *Effective March 15, 2024*, the CA SHOULD implement Multi-Perspective Domain Validation as described in this section. 
+- *Effective September 15, 2024*, the CA MUST implement Multi-Perspective Domain Validation as described in this section; however, the CA MAY proceed with certificate issuance if the Primary Domain Validation Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table.
+- *Effective March 15, 2025*, the CA MUST implement Multi-Perspective Domain Validation as described in this section; however, the CA MUST NOT proceed with certificate issuance if the Primary Domain Validation Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table and the "Defining Corroborating CAA Evidence" table.
+
+Even though the "# of Network Perspectives and Quorum Requirements" table permits issuance when some Network Perspectives do not corroborate the Primary Domain Validation Determination, the set of Network Perspectives that do corroborate the Primary Domain Validation Determination MUST be spread over two distinct continents for a CA to proceed with certificate issuance after March 15, 2025.
+
+Table: # of Network Perspectives and Quorum Requirements
+
+| Minimum # of Network Perspectives that MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination | Maximum # of Network Perspectives that MAY NOT corroborate the Primary Domain Validation Determination and Primary CAA Determination | Description |
+|---|---|---|
+| 1 | 1 | This configuration MUST NOT be used after 9/15/2025. |
+| 3 | 1 | This configuration MUST NOT be used after 9/15/2026. |
+| 4 | 2 | N/A |
+| N-2 | 2 | When the number of Network Perspectives used (i.e., “N”) is greater than or equal to (>=) six (6), a maximum of two (2) Network Perspectives MAY disagree with the Primary Domain Validation Determination. |
```

Comment:
I had to read this three times to realize why this row is talking about "6" (it's because the previous row documents 6 total perspectives: 4 which corroborate, and 2 which may disagree). I think this table would be helped by having a new left-most column with the total number of network perspectives.

Reply:
Just to transfer over a conversation from the google doc: Wayne Thayer mentioned that the leftmost column (which was originally in the table containing total number of perspectives) could be misinterpreted to be limiting the total number of perspectives a CA had deployed, not just how many they were using for that particular MPDV process. The resolution Ryan went with was to delete the total column, but I too had to read this table several times to interpret it correctly.

Reply:
That makes sense. I personally think labeling the column "# Network Perspectives used" would obviate that confusion as well.

Reply:
@dzacharo / @aarongable / @birgelee  - Would something this like be more helpful?

| # Network Perspectives used | Description of Configuration and Quorum Requirements | Permitted Use |
|--- |------------------------------ |------------------------------ |
| 2  | This configuration relies on two (2) Network Perspectives.<br><br>When this configuration is used, at least one (1) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | This configuration MUST NOT be used after 9/15/2025. |
| 3  | This configuration relies on three (3) Network Perspectives.<br><br>When this configuration is used, at least two (2) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination.  | This configuration MUST NOT be used after 9/15/2025. |
| 4  | This configuration relies on four (4) Network Perspectives.<br><br>When this configuration is used, at least three (3) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. |This configuration MUST NOT be used after 9/15/2026. |
| 5  | This configuration relies on five (5) Network Perspectives.<br><br>When this configuration is used, at least four (4) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | This configuration MUST NOT be used after 9/15/2025. |
| 6+ | This configuration relies on six (6) or more (i.e., "N") Network Perspectives.<br><br>When this configuration is used, at least N-2 Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | - |

(Pushed in https://github.com/ryancdickson/staging/commit/9dd72d52e0fb966e2ba17b3930fa5a3fe651c9a3)

Reply:
Just noticed my dates in the last column got mixed up, now fixed (Sorry!).